functions-and-operators/sequence-functions.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Sequence Functions
+summary: This document introduces sequence functions supported in TiDB.
+---
+
+# Sequence Functions
+
+The usage of sequence functions in TiDB are used with sequence objects that are created with the [`CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md) statement.
+
+| Function name | Feature description |
+| :-------------- | :------------------------------------- |
+| `NEXTVAL` or `NEXT VALUE FOR` | Returns the next value of the sequence |
+| `SETVAL` | Sets value of the sequence |
+| `LASTVAL` | Returns the last used value of the sequence |
```

Comment:
```suggestion
| `NEXTVAL()` or `NEXT VALUE FOR()` | Returns the next value of a sequence |
| `SETVAL()` | Sets the current value of a sequence |
| `LASTVAL()` | Returns the last used value of a sequence |
```

Reply:
I don't think we should add  `()` to `NEXT VALUE FOR`?

Reply:
Yes, you are right. I've updated the comment.